src/main/kotlin/sidechain/eth/util/Crypto.kt

```diff
@@ -4,26 +4,31 @@ import config.EthereumConfig
 import config.EthereumPasswords
 import notary.endpoint.eth.AmountType
 import org.web3j.crypto.Hash
-import org.web3j.protocol.admin.Admin
-import org.web3j.protocol.http.HttpService
 import org.web3j.utils.Numeric
+import org.web3j.crypto.Sign
 import java.math.BigInteger
 
-
 /**
  * Signs user-provided data with predefined account deployed on local Parity node
  * @param toSign data to sign
  * @return signed data
  */
 fun signUserData(ethereumConfig: EthereumConfig, ethereumPasswords: EthereumPasswords, toSign: String): String {
-    // TODO luckychess 26.06.2018 D3-100 find a way to produce correct signatures locally
-    val deployHelper = DeployHelper(ethereumConfig, ethereumPasswords)
 
-    val admin = Admin.build(HttpService(ethereumConfig.url))
-    admin.personalUnlockAccount(deployHelper.credentials.address, ethereumPasswords.credentialsPassword).send()
-    // TODO: signalize about error if unlock fails
+    val deployHelper = DeployHelper(ethereumConfig, ethereumPasswords)
```

Comment:
can we move deployHeper to the top level?

Reply:
Pass with params?

Reply:
Actually, we can even remove it now. Good point